esr_key_value_database_intrinsics.md

```diff
@@ -263,6 +273,11 @@ Returns the new iterator status. It aborts the transaction if `itr` wasn't retur
 If a key is found, the new status is `iterator_ok`. If not found, the new status is
 `iterator_end`. `kv_it_lower_bound` never returns `iterator_erased`.
 
+The size of the key and the size of the value at the iterator's new
+position, or zero if the iterator is at end, will be written to
+found_key_size and found_value_size. If found_key_size overlaps
+found_value_size, aborts the transaction.
```

Comment:
Or if they overlap with the range `[key, key + size)`.

Reply:
Need to update prototype above